src/test/regress/output/multi_outer_join.source

```diff
@@ -76,20 +115,17 @@ SELECT
 	min(t_custkey), max(t_custkey)
 FROM
 	multi_outer_join_third a LEFT JOIN multi_outer_join_right b ON (r_custkey = t_custkey);
-LOG:  join order: [ "multi_outer_join_third" ][ broadcast join "multi_outer_join_right" ]
```

Comment:
This is one of the few cases where we regress in the test coverage. Can't we make this test also pass with reference joins?